### PR TITLE
Drop obsolete bits

### PR DESCRIPTION
**Pull request by martinpitt**
* [x] image-refresh cirros

(Image refresh was just for testing the changed sink bits -- worked fine)

### CODE REVIEW

File: po-refresh [hunk 1]
```diff
@@ -49,19 +49,6 @@ def run(context, verbose=False, **kwargs):
...
-        # Make a tree with source and run "update-po" in it... outputs working directory
-        line = output("bots/make-source", "update-po")
-        work = os.path.abspath(line.strip())
-
-        # Do the various updates
-        cmd = ["make", "upload-pot", "clean-po", "download-po"]
-        if verbose:
-            sys.stderr.write("+ " + " ".join(cmd) + "\n")
-        subprocess.check_call(cmd, cwd=work)
-
```
Comment (jelly) [inline po-refresh:64]: Checked cockpit-certificates and it doesn't have translations set up so this seems fine.

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-03.
Log: https://logs.cockpit-project.org/logs/image-refresh-3534-20220621-202020/

File: tests-policy [hunk 2]
```diff
@@ -1,340 +0,0 @@
-#!/usr/bin/env python3
```
Comment (marusak) [inline tests-policy:1]: We still use this in rhel-7.9 branch.

Comment (martinpitt) [reply tests-policy]: Ah right, thanks for spotting! That commit doesn't backport at all, and I'm not in the mood of re-doing it, so I just dropped this commit.

**Review by marusak — APPROVED** [review]

**Pull request merged**